camply/cli.py

```diff
@@ -415,14 +415,11 @@ def campgrounds(
 equipment_id_argument = click.option(
     "--equipment-id",
     default=None,
-    help="""
-    Search for campsites campaitble with specific equipment categories. Going To
-    Camp uses equipment category IDs for filtering campsites by equipment. Every
-    recreation area has equipment categories unique to it.
-
-    Use `camply equipment-types --provider goingtocamp --rec-area <rec area id>`
-    to get a listing of equipment for an area.
-    """,
+    help="Search for campsites campaitble with specific equipment categories. Going To "
+    "Camp uses equipment category IDs for filtering campsites by equipment. Every "
+    "recreation area has equipment categories unique to it."
```

Comment:
We're missing one small bit of whitespace here after "unique to it." See the screenshot @ https://github.com/juftin/camply/pull/185#issuecomment-1371206884